pkg/controller/kieapp/test/crd_validation_test.go

```diff
@@ -184,9 +184,9 @@ func snippets(t *testing.T, file, yamlStr string) string {
 
 func TestJvmCrd(t *testing.T) {
 	crdYaml, _ := packr.New("deploy/crds", "../../../../deploy/crds").FindString("kieapp." + api.SchemeGroupVersion.Version + ".crd.yaml")
-	crdJson, _ := yaml.YAMLToJSON([]byte(crdYaml))
+	crdJSON, _ := yaml.YAMLToJSON([]byte(crdYaml))
```

Comment:
this test changes for JVM test are not related to default CPU settings, are they really needed?

Reply:
No, I'll remove this fix